src/components/FiltersBar/FiltersBar.tsx

```diff
@@ -277,7 +273,7 @@ export const FiltersBar: React.FC<ReturnType<typeof useFiltersBar> & FiltersBarP
               hasMarketing_eq: excludePaidPromotionalMaterialFilter ? !excludePaidPromotionalMaterialFilter : undefined,
               isExplicit_eq: excludeMatureContentRatingFilter ? !excludeMatureContentRatingFilter : undefined,
               categoryId_in: categoriesFilter,
-              languageId_eq: language as string,
+              languageId_eq: language === 'undefined' ? undefined : language,
```

Comment:
Why can't we use just `languageId_eq: language`?

Reply:
because Adam changed the language select a little bit and now has an `All languages` default option but this option can't be undefined in the `select` but it has to be undefined in the `languageId_eq`